src/misc.cpp

```diff
@@ -58,28 +67,61 @@ void InitializeCheats();
 void InitializeNPF();
 void InitializeOldNames();
 
+void RandomBytesWithFallback(std::span<uint8_t> buf)
+{
+#if defined(_WIN32)
+	auto res = BCryptGenRandom(nullptr, static_cast<PUCHAR>(buf.data()), static_cast<ULONG>(buf.size()), BCRYPT_USE_SYSTEM_PREFERRED_RNG);
+	if (res >= 0) return;
+#elif defined(__APPLE__) || defined(__NetBSD__) || defined(__FreeBSD__)
+	 arc4random_buf(buf.data(), buf.size());
```

Comment:
There's a space before `arc4random`, but more importantly: isn't this call as it stands now pointless? You will `buf` with random, and they you'll go into the fallback filling it with `InteractiveRandom`.

Reply:
Nice catch! Happens if you can't test a line of code :D